### PR TITLE
UHM-7022: unresolved label editing Prenatal Home encounter

### PR DESCRIPTION
**Pull request by mogoodrich**

### CODE REVIEW

File: configuration/pih/htmlforms/section-mch-referral.xml [hunk 1]
```diff
@@ -102,7 +102,7 @@
             <p class="radio">
                 <strong>
                     <label>
-                        CommCare&amp;nbsp;<uimessage code="mirebalais.login.username"/>
+                        CommCare&amp;nbsp;<uimessage code="pihcore.username"/>
```
Comment (lnball) [inline configuration/pih/htmlforms/section-mch-referral.xml:105]: Why not use "emr.user.username" which was already in the message properties?

It's a simple change.  If you want to use the new codes, go for it.

Comment (mogoodrich) [reply configuration/pih/htmlforms/section-mch-referral.xml:105]: Nice catch, I may have missed that, let me check.

Comment (mseaton) [reply configuration/pih/htmlforms/section-mch-referral.xml:105]: You can also use something from the authenticationui module.

Comment (mogoodrich) [reply configuration/pih/htmlforms/section-mch-referral.xml]: Not sure how I missed emr.user.username, I thought I had searched thoroughly... switched to that...

**Comment by lnball** [comment]
LGTM.  Simple ;)

**Pull request merged**